generate_export_options.rb

```diff
@@ -98,6 +98,7 @@ def export_method(mobileprovision_content)
 
 export_options = {}
 export_options[:method] = method unless method.nil?
+export_options[:uploadBitcode] = false
 
```

Comment:
Many apologies - I did not intend the latest commit to be part of the pull request. I haven't used forks and pull requests on github very much.

I am just testing the change locally. It would be optimal if the flag could be deduced from the ENABLE_BITCODE build setting.